modules/snooker/src/main/ts/ephox/snooker/api/Generators.ts

```diff
@@ -27,9 +31,7 @@ export interface SimpleGenerators extends Generators {
   readonly colgroup: () => SugarElement<HTMLTableColElement>;
 }
 
-export interface GeneratorsWrapper {
-  readonly cursor: () => Optional<SugarElement>;
-}
+export interface GeneratorsWrapper {}
```

Comment:
Is this still needed, if it's empty?

Reply:
It's still required, as it's a "base" interface.